src/crimson/osd/pg.cc

```diff
@@ -783,6 +783,10 @@ PG::submit_transaction(
   ceph_assert(log_entries.rbegin()->version >= projected_last_update);
   projected_last_update = log_entries.rbegin()->version;
 
+  for (auto &&entry: log_entries) {
```

Comment:
I doubt in current shape `&&` gives anything as `entry` is not `std::moved` below (so it's still an l-value reference IIUC).